.github/workflows/ci.yml

```diff
@@ -102,7 +103,6 @@ jobs:
       uses: codecov/codecov-action@d9f34f8cd5cb3b3eb79b3e4b5dae3a16df499a70
       with:
         files: ${{ matrix.covname }}
-        token: ${{ secrets.CODECOV_TOKEN }}
```

Comment:
Removed as we should not need a token for public repo

Reply:
🤷‍♂️